tests/Facebook/InstantArticles/Transformer/WPTransformerTest.php

```diff
@@ -6,7 +6,7 @@
  * This source code is licensed under the license found in the
  * LICENSE file in the root directory of this source tree.
  */
-namespace Facebook\InstantArticles\Transformer\CMS;
```

Comment:
This was an old structure when it was migrated from SDK.
Here it should match the folder structure.